spaceship/lib/spaceship/tunes/app_version.rb

```diff
@@ -609,6 +627,31 @@ def upload_trailer!(trailer_path, sort_order, language, device, timestamp = "05.
         setup_trailers
       end
 
+
+      # Uploads, app review attachments
+      #
+      # while submitting for review, ITC allow developers to attach file.
+      #
+      # File types allowed by Apple are: pdf, doc, docx, rtf, pages, xls, xlsx, numbers
+      # zip, rar, plist, crash, jpg, png, mp4 or avi.
+      #
+      # As of now fastlane support mp4, jpg, png and zip
```

Comment:
Is there more missing than just mime type that fastlane needs to support any more of these formats? (Ok, maybe not .numbers since it can be a directory or a flat file.)

Reply:
i have update the other PR
https://github.com/fastlane/fastlane/pull/14412